changelog.txt

```diff
@@ -11,12 +11,14 @@ next
 Added
 +++++
 
+
 - Add "fork" directive to enforce the execution of operations within a subprocess (#159).
 - Operation graph detection based on function comparison (#178).
 - Exceptions raised during operations always show tracebacks of user code (#169, #171).
 
 Changed
 +++++++
+- Raise warning when tag is not set and cannot be autogenerated for conditions and raise error with empty tag on graph detection (#195).
```

Comment:
This is a bit of a run-on sentence, it has too many "ands". The autogenerated part can be left out here.
```suggestion
- Raise a warning when a condition's tag is not set and raise an error if this occurs during graph detection (#195).
```